ALUM/ALUM/Components/PasswordInputFieldComponent.swift

```diff
@@ -0,0 +1,56 @@
+//
```

Comment:
Agree with Yash. `TextInputFieldComponent` is basically the same as `PasswordInputFieldComponent(isSecured: false, showEye: false)` so we should merge both into just a single component `TextInputFieldComponent` with secure handled by states.

Having a single component for both will help us in the long run as  we incorporate input validation and labels into the input component. 

We can ask the designers whether if we should add the `eye` icon to all password input components. If yes, just one state isSecured would be enough.

Reply:
Ok sounds good! I'll add those changes and let you know when I'm done!

Reply:
I added the changes, if possible take a look and let me know how it seems!